src/server/scripts/Outland/CoilfangReservoir/SerpentShrine/boss_hydross_the_unstable.cpp

```diff
@@ -160,9 +160,10 @@ struct boss_hydross_the_unstable : public BossAI
             }).Schedule(60s, GROUP_ABILITIES, [this](TaskContext)
```

Comment:
The rest also need to be incremented by 15 seconds. With the current state both the first and second mark are cast at the same time